cognee/infrastructure/databases/graph/neptune_driver/adapter.py

```
@@ -0,0 +1,1445 @@
+            - graph_name: The name of the graph to check for existence, defaults to 'myGraph'.
+              (default 'myGraph')
+
+        Returns:
+        --------
+
+            True if the graph exists, otherwise False.
+        """
+        pass
+
+
+    async def project_entire_graph(self, graph_name="myGraph"):
```

Ruff (0.12.2)

Redefinition of unused `project_entire_graph` from line 1172

(F811)